zscaler/README.md

```diff
@@ -0,0 +1,139 @@
+
+### ZIA DNS Logs
+
+1. From the ZIA console, go to **Administration** > **Nanolog Streaming Service**.
+2. Select the **Cloud NSS Feeds** tab. Then, click on **Add Cloud NSS Feed**.
+3. In the dialog box, enter or select the following values:
+   * Feed Name: `<YOUR_FEED_NAME>`
+   * NSS Type: `NSS for Firewall`
+   * SIEM Type: `Other`
+   * Batch Size: `16`
+   * API URL: `https://http-intake.logs.datadoghq.com/v1/input?ddsource=zscaler`
+   * HTTP headers:
+      * Key: `Content-Type`; Value: `application/json`
+      * Key: `DD-API-KEY`; Value: `<YOUR_DATADOG_API_KEY>`
+4. In the **Formatting** section, enter or select the following values:
+   * Log Type: `DNS logs`
+   * Feed Output Type: `JSON`
+   * Feed Escape Character: `\",`
+   * Feed output format:
+      ```
+      \{ "sourcetype" : "zscalernss-dns", "event" :\{"datetime":"%s{time}","user":"%s{login}","department":"%s{dept}","location":"%s{location}","reqaction":"%s{reqaction}","resaction":"%s{resaction}","reqrulelabel":"%s{reqrulelabel}","resrulelabel":"%s{resrulelabel}","dns_reqtype":"%s{reqtype}","dns_req":"%s{req}","dns_resp":"%s{res}","srv_dport":"%d{sport}","durationms":"%d{durationms}","clt_sip":"%s{cip}","srv_dip":"%s{sip}","category":"%s{domcat}","odeviceowner":"%s{odeviceowner}","odevicehostname":"%s{odevicehostname}"\}\}
+      ```
+5. Click **Save**.
+6. **Activate** your changes.
+
+### ZIA Tunnel Logs
+
+1. From the ZIA console, go to **Administration** > **Nanolog Streaming Service**.
+2. Select the **Cloud NSS Feeds** tab. Then, click on **Add Cloud NSS Feed**.
+3. In the dialog box, enter or select the following values:
+   * Feed Name: `<YOUR_FEED_NAME>`
+   * NSS Type: `NSS for Web`
+   * SIEM Type: `Other`
+   * Batch Size: `16`
+   * API URL: `https://http-intake.logs.datadoghq.com/v1/input?ddsource=zscaler`
+   * HTTP headers:
+      * Key: `Content-Type`; Value: `application/json`
+      * Key: `DD-API-KEY`; Value: `<YOUR_DATADOG_API_KEY>`
+4. In the **Formatting** section, enter or select the following values:
+   * Log Type: `Tunnel`
+   * Feed Output Type: `JSON`
+   * Feed Escape Character: `\",`
+   * Feed Output Format:
+      ```
+      \{ "sourcetype" : "zscalernss-tunnel", "event" : \{"datetime":"%s{datetime}","Recordtype":"%s{tunnelactionname}","tunneltype":"IPSEC IKEV %d{ikeversion}","user":"%s{vpncredentialname}","location":"%s{locationname}","sourceip":"%s{sourceip}","destinationip":"%s{destvip}","sourceport":"%d{srcport}","destinationport":"%d{dstport}","lifetime":"%d{lifetime}","ikeversion":"%d{ikeversion}","spi_in":"%lu{spi_in}","spi_out":"%lu{spi_out}","algo":"%s{algo}","authentication":"%s{authentication}","authtype":"%s{authtype}","recordid":"%d{recordid}"\}\}
+      ```
+5. Click **Save**.
+6. **Activate** your changes.
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `zscaler` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
```

Comment:
Should we say the integration doesn't include any metrics (the metadata.csv doesn't list any)?

Reply:
Good catch - updated this snippet in 6d0f6d3760e14bc5f33ba225b58cf2fdc189cc3e

Reply:
Thanks! Can the metadata.csv file also be removed, since it's empty? Otherwise everything looks good!

Reply:
Thanks! Can the metadata.csv file also be removed, since it's empty? Otherwise everything looks good!